src/style/card.css

```
@@ -0,0 +1,81 @@
+.card {
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__title {
+  margin: 40px 0 0;
```

just margin-top. Update for similar cases 
```suggestion
  margin-top: 40px;
```